docs/interaction-area.html

```diff
@@ -22,10 +22,6 @@
       var cld = cloudinary.Cloudinary.new({cloud_name: 'demo'});
       var options = {fluid: true};
 
-      player = cld.videoPlayer('player', options);
-
-      player.source('https://res.cloudinary.com/eran2903/video/upload/eo_8,w_270,q_auto:best,f_auto/production_ID_3888264_fuuo7c.mp4');
-
       var interactionAreas = [
```

Comment:
how about we add identifiers to each interaction are so that we can recognize it by name/id instead/in addition to index?

Reply:
will do